fetch.bs

```diff
@@ -4422,6 +4538,14 @@ steps:
 
  <li><p><a>Upgrade <var>request</var> to a potentially trustworthy URL, if appropriate</a>.
 
+ <li>
+  <p>Optionally, run <a>upgrade an HTTP request</a> algorithm on <var>request</var>, if appropriate</a>.
+
+  <p class=note>HTTPS upgrading only applies to requests with <a>HTTP(S) scheme</a>s, but it's done
+  in <a>main fetch</a> instead of <a>HTTP fetch</a> to ensure that <a>upgrade a mixed content
```

Comment:
`<a>` elements are always on a single line, so you need to wrap before (and likely after because this is a long one).

Reply:
Done